float64 input_temp
string conversion_type  # Cel_to_Far o Far_to_Cel
---
float64 converted_temp
